pom.xml

```diff
@@ -159,7 +159,7 @@
     <jmh.version>1.26</jmh.version>
     <audienceannotations.version>0.13.0</audienceannotations.version>
     <clp-ffi.version>0.4.3</clp-ffi.version>
-
+    <aws.sdk.version>2.20.83</aws.sdk.version>
```

Comment:
technically speaking we dont need aws sdk if we dont enable the plugin. we can add a comment here (or not, i dont think that's too biggy)